### ISSUE TITLE
Icons in color can have low contrast

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6035]
Depending on whether Scratch Addons is in light or dark mode, the colored icons such as [heart](https://github.com/ScratchAddons/ScratchAddons/blob/master/images/icons/heart.svg) and [star](https://github.com/ScratchAddons/ScratchAddons/blob/master/images/icons/star.svg) can be hard to read. This isn't a huge deal when they are next to text since there, they aren't essential, but when you don't have any text, that's when it's an accessibility issue. In the Scratch Messaging popup, some of these icons are on their own and they don't adapt to the page background to ensure contrast against the background.

We should find a way to make at least the important ones change color based on the selected theme so they aren't too hard to make out. However, if we make this change to a few, it might be good to make this change to all the icons with color.

Note: I plan to remove the color from category icons in the category selection pane (see [`daedf06`](https://github.com/ScratchAddons/ScratchAddons/commit/daedf065acb20c48cfb819a4cd94e04997d61b86#diff-bbf60d323e1ba33d53e0687bf37cae858423c957ff1f118d6dc58b7b10bf6eceL90-L92)), so no need to modify those.

**Comment by Secret-chest** [comment]
Would filling them in be good?

**Comment by DNin01** [comment]
> Would filling them in be good?

It'd probably make them a little better, but not perfect.

**Comment by DNin01** [comment]
> Would filling them in be good?

Actually, that is worth thinking about, but I think they would still need a color change.

All we should need to do is just make the heart brighter in dark mode and the star darker in light mode and they should be good. Nothing major.
